config.py

```diff
@@ -177,9 +179,16 @@ def close(self):
             self.save()
             self.defaults = None
 
+        def set_shutdown(self):
+            self.__in_shutdown = True
+
+        def shutting_down(self) -> bool:
+            return self.__in_shutdown
+
```

Comment:
May as well make this a property so access isnt `config.shutting_down()`

Reply:
Done